src/Mgate/PersonneBundle/Resources/translations/person.fr.yml

```diff
@@ -0,0 +1,155 @@
+generic:
```

Comment:
Adding translations is nice, but as long as that project is mainly french, translations keys should be in french. It makes main developers work much easier.

I am sorry to say that because I know it has been a big work for you, but would you mindchanging your work to  match the already existing conventions. You can have an example in [suivi translations](https://github.com/n7consulting/Incipio/blob/master/src/Mgate/SuiviBundle/Resources/translations/suivi.fr.yml)